apps/mobile/src/app/(home)/activity/components/activity-widget.tsx

```diff
@@ -17,8 +17,9 @@ interface ActivityWidgetProps {
 export function ActivityWidget({ activity, isLoading, onPressHeader }: ActivityWidgetProps) {
   const theme = useTheme<Theme>();
 
+  const hasActivity = activity && activity.length > 0;
   // TODO LEA-1726: handle  loading & error states
-  if (isLoading || !activity) return null;
+  if (isLoading || !hasActivity) return null;
```

Comment:
This will likely need an empty state design, because once loading state is implemented, showing a loading state then nothing would be strange. Worth creating a ticket for the design team to take care of this.

Reply:
Thanks 👍  They have some empty states we need to build I linked in LEA-1726